sbncode/EventGenerator/MeVPrtl/MeVPrtlTestRayTrace_module.cc

```diff
@@ -118,8 +118,8 @@ void evgen::ldm::MeVPrtlTestRayTrace::analyze(const art::Event& evt)
   while (1) {
     simb::MCFlux kaon = fGenTool->GetNext();
 
-    evgen::ldm::KaonParent kaonp(kaon);
-    bool is_kaon = kaonp.kaon_pdg != 0;
+    evgen::ldm::MesonParent kaonp(kaon);
```

Comment:
The code is fine, but for consistency the variables and the message would be better changed to "meson" instead of "kaon".